### ISSUE TITLE
Does not get as red as it should

### ISSUE THREAD

**Issue by steelstring94** [kotelnik/plasma-applet-redshift-control#51]
I have it set to go to 1500K at night.  It does not do that on its own, though.  If I mouse over the system tray lightbulb, it will say it's at 1500K, but if I adjust it even one tick with my scrollwheel, or click it off and back on again, the screen will suddenly get much redder.  It's as though it's unable to redden past a certain point by itself.

**Comment by jkufner** [comment]
Try to check if there is only one instance of redshift running in the background. Maybe you have an unconfigured instance running for some weird reason.

**Comment by steelstring94** [comment]
You are correct, two instances of Redshift are running somehow, but I have no idea why.
